app/src/main/java/de/westnordost/streetcomplete/ApplicationConstants.java

```diff
@@ -29,6 +29,9 @@
 	public static final String LUTIM_INSTANCE = "https://lut.im/"; // must have trailing /
 	public static final Integer LUTIM_DELETE_AFTER_DAYS = 365 / 2; // half a year
 
+	public static final String COMPLETE_INSTANCE = "https://ent8r.lima-city.de/CompleteStreetComplete/";
+	public static final String COMPLETE_AUTH = new String(new char[]{100,101,109,111,117,115,101,114,58,49,50,51});
```

Comment:
Huh? Is this the "API key" for your server?

So…
* why use this char thing, and not a simple ascii api key (see the ones GitHub or so provides)
* and, secondly, maybe allow one to specify the API key at built time. Don't know how it is handled for the mapzen key, but usually API keys should not be public in (GitHub) repositories.

Reply:
> Don't know how it is handled for the mapzen key

The mapzen API key is also a char[] and if those keys would not be public, nobody could build his own version of StreetComplete and they would need to get their own mapzen API key...